react-es6-popup/.babelrc

```diff
@@ -1,11 +1,7 @@
 {
   "presets": [
-    "es2015",
+    "env",
```

Comment:
By default, it [looks like](http://babeljs.io/docs/plugins/preset-env/) this will transform everything for general web compatibility, including IE and Safari. I think that would be overkill for a browser extension. What if we limited it to Firefox (and maybe Chrome too), like this?

```json
{
  "presets": [
    ["env", {
      "targets": {
        "firefox": 57
      }
    }]
  ]
}
```